app/stylesheet/menu-colors.scss

```diff
@@ -1,4 +1,4 @@
-$rail-primary-bg: $ui-background; // carbon--gray-90
+$rail-primary-bg: $carbon--gray-100; // carbon--gray-100
```

Comment:
what was ui-background before?

Reply:
ui-background is defined by the current carbon theme

@kavyanekkalapu does that change really belong upstream or should this be in the plugin?

Reply:
I believe gray-90 was an intentional choice for upstream to differentiate from downstream.

Reply:
Yeah, my point was... your change switches the upstream UI from using a theme-dependent color to using a specific color instead.

So... should this even be an upstream change? (Different color.) 
And should we not be using the theme's background color for the menu?

(I don't care about gray 90 vs gray 100, but the change doesn't make sense without knowing what it's for.)

Reply:
Reverted the color change @himdel @Fryguy